### DISCUSSION TITLE
欢迎有志之士来讨论

### DISCUSSION THREAD

**Discussion by 0-8-4** [0-8-4/miui-auto-tasks#135] [Announcements]
![IMG_1138](https://github.com/0-8-4/miui-auto-tasks/assets/31277785/f64ac48f-6847-41e7-a656-44868b8f4766)
# 懂得都懂

**Comment by 0-8-4** [comment]
@PianCat @Night-stars-1 @TardisLX

**Comment by Adlyq** [comment]
过期了？

**Reply by 0-8-4** [reply]
毕竟互联网不是法外之地，当然要设置有效期，看最新的回复

**Comment by 0-8-4** [comment]


**Reply by ni-feng-le** [reply]
链接失效了

**Comment by ni-feng-le** [comment]


**Reply by 0-8-4** [reply]
解了

**Comment by ni-feng-le** [comment]
![Uploading IMG_20231213_194745_481.jpg…]()

**Comment by 0-8-4** [comment]
<img width="359" alt="image" src="https://github.com/0-8-4/miui-auto-tasks/assets/31277785/230bd86c-baef-48dc-b017-10fbe896770b">

#   最新

**Reply by ni-feng-le** [reply]
群呢，咋又不见了

**Comment by cocoyamnut** [comment]
已过期

**Comment by Linnianci** [comment]
请问群组还可以加入吗

**Comment by xiaoshengren2** [comment]
请发一个最新的群，谢谢

**Comment by jshir700** [comment]
请问群组是否还可以添加呀~
